api/integration/validation_test.go

```diff
@@ -64,7 +64,7 @@ func TestValidateHandlerInvalidAddress(t *testing.T) {
 	api.Geocode = mock.Geocoder{}
 
 	// Pass a bad address
-	location := readTestData(t, "../testdata/bad-address.json")
+	location := readTestData(t, "../testdata/nonstandardized-address.json")
```

Comment:
This is great for testing the nonstandard addresses to make sure validation is still working as expected for those.
Please also write a test to consider when a bad address (such as what was in bad-address.json) is attempted.

Reply:
@kilbergr That's already covered by the existing tests; take a look at `TestValidateHandlerBadEntity`, `TestCanNotValidateSomethingElse`.  The main problem here was that the implementation was changed incorrectly and the test was changed to match the incorrect behavior.

Reply:
OK I see what you're saying. Thanks for addressing.